packages/trpc/server/routers/viewer/organizations/adminUpdate.handler.ts

```diff
@@ -49,13 +50,33 @@ export const adminUpdateHandler = async ({ input }: AdminUpdateOptions) => {
     data.metadata = mergeMetadata({
       // If we save slug, we don't need the requestedSlug anymore
       requestedSlug: undefined,
-      ...input.metadata,
     });
   }
 
-  const updatedOrganisation = await prisma.team.update({
-    where: { id },
-    data,
+  const updatedOrganisation = await prisma.$transaction(async (tx) => {
+    const updatedOrganisation = await prisma.team.update({
```

Comment:
For it to be considered in a transaction I think it needs to use `tx`
```suggestion
    const updatedOrganisation = await tx.team.update({
```